deployment/nuget-keys.md

```diff
@@ -3,115 +3,105 @@ title: Restoring NuGet Packages in CI
 page_title: Restoring NuGet Packages in CI
 description: Get started with Telerik UI for Blazor and use NuGet Keys to authenticate with the Telerik NuGet server and restore packages in your CI or desktop environment.
 slug: installation/nuget-keys
-position: 7
+position: 10
 ---
 
 # Restoring NuGet Packages in Your CI Workflow
 
-This article provides an overview of the most popular ways for using token-based authentication to restore Telerik NuGet packages in your Continuous Integration (CI) workflow.
+This article describes how to use token-based authentication for the Telerik NuGet feed. You will learn how to create and use NuGet API keys to restore Telerik NuGet packages in your Continuous Integration (CI) workflow.
 
-The Telerik NuGet server allows you to authenticate by using two methods:
+The [Telerik NuGet server]({%slug installation/nuget%}) provides two ways to authenticate:
 
-* Basic authentication by providing your Telerik user name and password.
-* Token-based authentication by providing a NuGet Key.
+* Basic authentication with your Telerik username and password.
+* Token-based authentication with a NuGet API key.
 
-When you need to restore Telerik NuGet packages as part of your CI, using NuGet keys is the more secure way to authenticate. This method does not require you to provide your Telerik username and password anywhere in the CI workflow.
+When you need to restore the [Telerik NuGet packages]({%slug getting-started/what-you-need%}#nuget-packages) as part of your CI, a NuGet API key is the more secure way to authenticate. This method does not require you to provide your Telerik credentials anywhere in the CI workflow. A NuGet Key has a limited scope and can be used only with the Telerik NuGet server. If any of your NuGet keys is compromised, you can quickly delete it and create a new one.
 
-Unlike your Telerik credentials, a NuGet Key has a limited scope and can be used only with the Telerik NuGet server. If any of your NuGet keys is compromised, you can quickly delete it and create a new one.
 
 ## Generating NuGet Keys
 
-1. Go to the [**Manage NuGet Keys**](https://www.telerik.com/account/downloads/nuget-keys) page in your Telerik account.
+1. Go to page [**Manage NuGet Keys**](https://www.telerik.com/account/downloads/nuget-keys) in your Telerik account.
```

Comment:
Placing the context first is a good practice, especially in this case where you haven't been explicit that they need to log into their account first.

```suggestion
1. In your Telerik account, go to the [**Manage NuGet Keys**](https://www.telerik.com/account/downloads/nuget-keys) page.
```